File: 2491-smallest-even-multiple/2491-smallest-even-multiple.cpp
class Solution {
public:
    int hcf(int a,int b)
    {
        if(b==0) return a;
        return hcf(b,a%b);
    }
    int smallestEvenMultiple(int n) {
        int hcf1=hcf(2,n);
        return (2*n)/hcf1;
    }
};

File: 2491-smallest-even-multiple/2491-smallest-even-multiple.java
class Solution {
    public int hcf(int a, int b)
    {
        if(b==0) return a;
        return hcf(b,a%b);
    }
    public int smallestEvenMultiple(int n) {
        int hcf=hcf(2,n);
        return (2*n)/hcf;
    }
}